src/xrServerEntities/xrServer_Objects_ALife.h

```diff
@@ -168,6 +168,7 @@ class CSE_ALifeObject : public CSE_Abstract, public CRandom
     virtual u32 ef_weapon_type() const;
     virtual u32 ef_detector_type() const;
 #ifdef XRGAME_EXPORTS
+    virtual bool is_spawn_supplies_flag_set(LPCSTR, LPCSTR);
```

Comment:
Please, use pcstr and name the parameters:
```suggestion
    virtual bool is_spawn_supplies_flag_set(pcstr value, pcstr flag);
```

Reply:
Don't look at that spawn supplies below :)